src/www/app/outline_server_repository/server.ts

```diff
@@ -19,7 +19,7 @@ import {Server} from '../../model/server';
 import {NativeNetworking} from '../net';
 import {Tunnel, TunnelStatus, ShadowsocksSessionConfig} from '../tunnel';
 
-import {accessKeyToShadowsocksSessionConfig} from './access_key_serialization';
+import {staticKeyToShadowsocksSessionConfig} from './access_key_serialization';
 
 // PLEASE DON'T use this class outside of this `outline_server_repository` folder!
 
```

Comment:
I'm debating whether we should rename this to `Service`, at least in the code. We can align the UI later. That should clarify some of the concepts. In particular, the Server or Shadowsocks Service is determined dynamically for the Outline Service

Reply:
I tried to rename server to service at one point and it was a HUGE change. Is it alright if I keep it as Server for now and promise to change everything at the end?

Reply:
Ah, ok. Hopefully the refactoring tools will help.